LocalStorage/LocalStorageHelpers/RealmStorageHelper.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  RealmStorageHelper.swift
+//  WeatherDemoApp
+//
+//  Created by ahmed on 05/04/2024.
+//
+
+import RealmSwift
+import Combine
+
+final class RealmStorageHelper: LocalStorageProtocol {
+    private var realm = try! Realm()
+    
+    private var notificationToken: NotificationToken!
+    private var citiesObservervable: Results<CityRealmObject>!
+    
+    /// outputs
+    let cities = PassthroughSubject<Result<[LocalStorageCity], LocalStorageError>, Never>()
```

Comment:
I believe this is something unrelated to the object responsibility and should be removed outside.